tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +44,7 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
We could also check the length of the vendor's inventory after removing an item to make the test more robust